src/Vgrep/Environment/Config.hs

```diff
@@ -2,62 +2,92 @@
 {-# LANGUAGE TemplateHaskell #-}
 module Vgrep.Environment.Config where
 
-import Control.Lens.Compat
-import Control.Monad.IO.Class
-import Data.Maybe
-import Data.Monoid
-import Graphics.Vty.Image
-
-import Vgrep.Environment.Config.Monoid
-import Vgrep.Environment.Config.Sources
+import           Control.Lens.Compat
+import           Control.Monad.IO.Class
+import           Data.Map.Strict        (Map)
+import qualified Data.Map.Strict        as M
+import           Data.Maybe
+import           Data.Monoid
+import           Graphics.Vty.Image
+    ( Attr
+    , blue
+    , bold
+    , defAttr
+    , green
+    , standout
+    , withBackColor
+    , withForeColor
+    , withStyle
+    )
+
+import           Vgrep.Command
+import           Vgrep.Environment.Config.Monoid
+import           Vgrep.Environment.Config.Sources
+import qualified Vgrep.Key                        as Key
 
 
 --------------------------------------------------------------------------
 -- * Types
 --------------------------------------------------------------------------
 
 data Config = Config
-    { _colors :: Colors
+    { _colors      :: Colors
     -- ^ Color configuration
 
-    , _tabstop :: Int
+    , _tabstop     :: Int
     -- ^ Tabstop width (default: 8)
 
-    , _editor :: String
+    , _editor      :: String
     -- ^ Executable for @e@ key (default: environment variable @$EDITOR@,
     -- or @vi@ if @$EDITOR@ is not set)
 
+    , _keybindings :: Keybindings
+
     } deriving (Eq, Show)
 
 data Colors = Colors
-    { _lineNumbers :: Attr
+    { _lineNumbers   :: Attr
     -- ^ Line numbers (default: blue)
 
     , _lineNumbersHl :: Attr
     -- ^ Highlighted line numbers (default: bold blue)
 
-    , _normal :: Attr
+    , _normal        :: Attr
     -- ^ Normal text (default: terminal default)
 
-    , _normalHl :: Attr
+    , _normalHl      :: Attr
     -- ^ Highlighted text (default: bold)
 
-    , _fileHeaders :: Attr
+    , _fileHeaders   :: Attr
     -- ^ File names in results view (default: terminal default color on
     -- green background)
 
-    , _selected :: Attr
+    , _selected      :: Attr
```

Comment:
OTOH, it's not _that_ confusing, and I rather have `stylish-haskell` add gratuitous spaces, than disabling aligning of records altogether...